locations/spiders/fire_and_rescue_nsw_au.py

```diff
@@ -0,0 +1,39 @@
+from scrapy.spiders import SitemapSpider
+
+from locations.categories import apply_category
+from locations.google_url import extract_google_position
+from locations.items import Feature
+
+
+class FireAndRescueNSWAUSpider(SitemapSpider):
+    name = "fire_and_rescue_nsw_au"
+    allowed_domains = ["www.fire.nsw.gov.au"]
+    sitemap_urls = ["https://www.fire.nsw.gov.au/feeds/sitemap.xml"]
+    sitemap_rules = [(r"^https:\/\/www\.fire\.nsw\.gov\.au\/page\.php\?id=9210&station=\d+", "parse")]
+    no_refs = True
```

Comment:
Can we use the url as the ref?

Without a ref the output ids aren't stable.

Reply:
Done :)

Reply:
Thank you